actions/UploadAction.php

```diff
@@ -110,6 +110,12 @@ public function init()
     public function run()
     {
         if (Yii::$app->request->isPost) {
+            $imageDir = '/' . date('d-m-Y') . '/';
```

Comment:
- Will be better to have here a function call to can generate own rule for directory name
- Need a option to turn on/off this behavior.